drivers/can/can_mcp2515.c

```diff
@@ -363,8 +366,7 @@ static int mcp2515_set_timing(const struct device *dev,
 	const uint8_t rx0_ctrl = BIT(6) | BIT(5) | BIT(2);
 	const uint8_t rx1_ctrl = BIT(6) | BIT(5);
 
-	__ASSERT((timing->sjw >= 1) && (timing->sjw <= 4),
-		 "1 <= SJW <= 4");
+	__ASSERT(timing->sjw <= 4, "1 <= SJW <= 4");
```

Comment:
```suggestion
```

Reply:
Why removing this check?

Reply:
I suggested to move it into the section where `dev_data->sjw` actually gets written, into the `if (timing->sjw != CAN_SJW_NO_CHANGE)` block
but it's ok the way it is